CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.0.0]
+### Changed
+- Update InfuraJsonRpcSupportedNetwork with supported networks ([#69](https://github.com/MetaMask/eth-json-rpc-infura/pull/69))
```

Comment:
We should note here that it's a breaking change, and explain why it is breaking. e.g.
```suggestion
- **BREAKING:** Update `InfuraJsonRpcSupportedNetwork` with supported networks ([#69](https://github.com/MetaMask/eth-json-rpc-infura/pull/69))
  - The following networks have been removed from this type because they are no longer supported by Infura:
    - 'ropsten'
    - 'rinkeby'
    - 'kovan'
    - 'eth2-beacon-mainnet'
    - 'optimism-kovan'
    - 'arbitrum-rinkeby'
```

Reply:
This is understandable to me.